### DISCUSSION TITLE
See Vaultwarden Version on a public instance as a user?

### DISCUSSION THREAD

**Discussion by ghost** [dani-garcia/vaultwarden#5211] [Q&A] [answered]
Is it possible as a user to see the Vaultwarden version (1.xx.x) on a Vaultwarden public instance, or do you have to trust the operator?
At the moment, I think you can only see the Web Vault version when you log in.

**Comment by BlackDex** [comment, marked as answer]
There are two ways (Unless someone blocked it)
- `https://vw.domain.tld/api/version`
  This could either return the following:
  - `1.32.4-cdfdc6ff`
  - `1.32.5`

- `https://vw.domain.tld/api/config`
  Should have the `gitHash` value filled with the commit used to build Vaultwarden.
  Which should be `cdfdc6ff` for the current latest version.

**Reply by ghost** [reply]
Thank you for the quick response. That worked for me.
